### PR TITLE
Leaderboard: hit the API

### PR DESCRIPTION
**Pull request by ntzwilly**
# Hit the API
## Description

- Create a new game by using the API.
- Implement the "Refresh" button (receiving data from the API and parsing the JSON).
- Implement the form "Submit" button (sending data to the API).
- Use async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-use-before-define */
```
Comment (Shaher-11) [inline src/api.js:2]: Kindly remove the eslint disable

File: src/index.js [hunk 2]
```diff
@@ -1,51 +1,51 @@
-/* eslint-disable no-plusplus */
+/* eslint-disable import/no-cycle */
```
Comment (Shaher-11) [inline src/index.js:1]: Kindly remove the eslint disable

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @ntzwilly,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Check the comments below:point_down:


>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

File: src/api.js (hunk 1 above)
Comment (ntzwilly) [reply src/api.js]: Thank you, I removed the eslint disable

**Comment by Davidosky007** [comment]
# Hi @ntzwilly 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
job-welldone-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**